community/kernel/src/test/java/org/neo4j/kernel/impl/newapi/MockStore.java

```diff
@@ -25,9 +25,9 @@
 
 import org.neo4j.collection.primitive.Primitive;
 import org.neo4j.collection.primitive.PrimitiveLongObjectMap;
+import org.neo4j.internal.kernel.api.*;
```

Comment:
Star imports not allowed, I thought that checkstyle caught those.

Reply:
It does apparently https://build.neohq.net/viewLog.html?buildId=2383763&buildTypeId=PullRequests_BranchPackaging&tab=buildLog&_focus=4287